sdk/storage/azure-storage-common/inc/azure/storage/common/internal/constants.hpp

```diff
@@ -18,6 +18,7 @@ namespace Azure { namespace Storage { namespace _internal {
   constexpr static const char* HttpHeaderContentType = "content-type";
   constexpr static const char* HttpHeaderContentLength = "content-length";
   constexpr static const char* HttpHeaderContentRange = "content-range";
+  constexpr static const char* DefaultSasVersion = "2020-02-10";
```

Comment:
Since blob sdk March release is a beta release, I have to restore this line of code so that the last GA release still works. I'll remove this line after April release.